### PR TITLE
Removed javax.xml.bind.attachment import package - Fix ESBJAVA-4778

### PR DESCRIPTION
**Pull request by jagathsisira**
Updated cxf version to 3.0.9

### CODE REVIEW

File: cxf/3.0.9.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+        <dependency>
+            <groupId>org.apache.cxf</groupId>
+            <artifactId>cxf-rt-frontend-jaxws</artifactId>
+            <version>${version.cxf}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.fastinfoset</groupId>
+            <artifactId>FastInfoset</artifactId>
+            <version>1.2.13</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: Add versions as property as it will be easy to maintain.

File: cxf/3.0.9.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.woodstox</groupId>
+            <artifactId>woodstox-core-asl</artifactId>
+            <version>4.4.1</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: same

File: cxf/3.0.9.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,478 @@
...
+        <dependency>
+            <groupId>org.apache.neethi</groupId>
+            <artifactId>neethi</artifactId>
+            <version>3.0.3</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.ws.wso2</groupId>
+            <artifactId>spring.framework</artifactId>
+            <version>3.2.9.wso2v1</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: same

File: cxf/3.0.9.wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,478 @@
...
+        <dependency>
+            <groupId>org.apache.ws.xmlschema</groupId>
+            <artifactId>xmlschema-core</artifactId>
+            <version>2.1.0</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.neethi</groupId>
+            <artifactId>neethi</artifactId>
+            <version>3.0.3</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: same

File: cxf/3.0.9.wso2v1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+        <dependency>
+            <groupId>org.codehaus.woodstox</groupId>
+            <artifactId>woodstox-core-asl</artifactId>
+            <version>4.4.1</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ws.xmlschema</groupId>
+            <artifactId>xmlschema-core</artifactId>
+            <version>2.1.0</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: same

File: cxf/3.0.9.wso2v1/pom.xml [hunk 6]
```diff
@@ -0,0 +1,478 @@
...
+                                <transformer
+                                        implementation="org.apache.maven.plugins.shade.resource.XmlAppendingTransformer">
+                                    <resource>META-INF/cxf/java2wsbeans.xml</resource>
+                                </transformer>
+                                <transformer implementation="org.apache.cxf.maven.CXFAllTransformer"/>
+                            </transformers>
+                        </configuration>
+                    </execution>
+                </executions>
+                <dependencies>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: add this to dependencies list on above. Having 2 places can be error prone,

File: cxf/3.0.9.wso2v1/pom.xml [hunk 7]
```diff
@@ -0,0 +1,478 @@
...
+                    <dependency>
+                        <groupId>org.apache.cxf</groupId>
+                        <artifactId>cxf-buildtools</artifactId>
+                        <version>2.2.12</version>
```
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: use maven property

File: cxf/3.0.9.wso2v1/pom.xml (hunk 2 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 1 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 5 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 4 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 3 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 6 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: This dependency is for plugin execution. Therefore didn't change it.

File: cxf/3.0.9.wso2v1/pom.xml (hunk 7 above)
Comment (jagathsisira) [inline cxf/3.0.9.wso2v1/pom.xml]: Fixed

File: cxf/3.0.9.wso2v1/pom.xml (hunk 6 above)
Comment (manoj-kumara) [inline cxf/3.0.9.wso2v1/pom.xml]: Yeah, I missed that.

**Pull request merged**